cylc/uiserver/resolvers.py

```diff
@@ -89,11 +202,37 @@ def _return(message):
             message
         ]
 
+    @classmethod
+    async def clean(cls, workflows, args, workflows_mgr, log, executor):
+        """Calls `init_clean`"""
+        response = []
+
+        # Convert Schema options → cylc.flow.workflow_files.init_clean opts:
+        try:
+            opts = _schema_opts_to_api_opts(args)
+        except Exception as exc:
+            return cls._error(exc)
+        # Hard set remote timeout.
+        opts.remote_timeout = "600"
```

Comment:
Very minor point but this defaults to 120 in cylc flow. Wonder if it would be worth making them consistent, for documentation purposes mainly.
I think 600 may seem like the better timeout than 120, since nfs and file deletions can perhaps take a while?

Reply:
In the CLI 120 is the default, but the user can over-ride it.
In the UI I haven't exposed it to the user, so I think it's reasonable to set it longer.

Reply:
I can't remember the reason, but I think @oliver-sanders and I agreed no to.